app/Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 
 gem 'sinatra', '>= 2.1.0'
-gem 'smartcar', '>= 1.0.7'
+gem 'smartcar', git: 'https://github.com/smartcar/ruby-sdk'
```

Comment:
lets deploy new version and just use it from rubygems instead of directly looking at git.